### ISSUE TITLE
Multiple List concat operators are not seen as the same operator.

### ISSUE THREAD

**Issue by nojaf** [fsprojects/fantomas#2472]
Issue created from [fantomas-online](https://fsprojects.github.io/fantomas-tools/#/fantomas/preview?data=N4KABGBEDGD2AmBTSAuKEMYDaIC5gEMssARRaLAZzAF4wBLAOwAtEAne3SgGTAAEG9ALYAHKt2oDouWGwn8GjSrgKNoiAGpF5AoXmY6GWAGL1EWeDwUi2sEYcQA3RI1yHlBXPWhashxojKiPAAKgCeIogSADps0Yyx8ZiYOPhEpORUtIqsHFy8ANSCouLURdKy8kVMHmqa2mVgergGjfQmZhZWRTZ2VWBOLm6NHl4%2BDWBFAUGhEVGUiYyQADTgUJR4XowA5pSoYADaaxCgyVAAJLhz%2B5BMuCvHGJDnjkQArshoR2cQAAyrPygAElGEhXABlegAL2QAJ%2BABZHhAALqPAC%2BcJOSIuV0iNzuDzOz1eWA%2B%2B2%2BPwAjJjkpAALIEAAe3CYURc2xahKpACZftjUZgMY9TrTLtc0JAXPAAPIAMxZAXBVxwXMwxPen0O2Ig3JpaoAoqC5QrYdqoFhZZB%2BeiaSK1WK8RKAEawWBYVVPF4a8lmgDMeqeII2bFwpkYRAAcogAO5YVkezC4Ngfa2C23Y564zWQF1uhMXElkr5m%2BEBiCQcEiAjqABCiFlskQAAUCGwCM12PmIEmU2cBRghZg7Z6szdc%2B6A%2BrSZqKWcAKxlqCV6uIOsNthRWDR9jQAgbEGOWC7rywJaLnuIVMD9NEh3Z8f5qdFrWAgBsi4rVdr9cbAFURJEbC7vujCHse9Cnl2YCykQGxXhAg4YMO5Z3mOroThmXrTj6gIAOwfsu37rogADCWB7pQJGnsoyYVGwUEwVQl59jawqYaOzroY%2BWHPrOyQABwEV%2Bq4-hudKIEITqdoujFwSxaZsbeHFQA%2Bk48TOZoAJxCSua6NlRWCQTJsHMck-YITeorKTmXFqYWGmApS-xmp%2BK4AIKyrg7BUUIQgEFBF7wWAiFYkp4oqbZmH2ThVLUi5hEicR4ISd4bpGWasmmZg5nBZZ9rWapUXesWjm6vFwkeV5bDJUIqWGWeZqBfJ16KVZ4U2XmdnFS%2BVL%2BuV7m2G8oJkHGtVVQFyZZRgOUhWAyE4u1BJddhJVUqWLkMoyQKyiErCMOCzCyLgADq9DwJyi58s1Fmtfli2uNx0WrWclILhtTLbbtLj6kxB1Had53MFBr5XWZrFDux933MtvFmpS77vVtjCyvQjLSoBniyPqjI2IElAQQ1gL8aD2Xg0hkOOlAS1FStPUvfhiMAErkLI8AAxdJYk9NZOhW1lO3A9MMOVSglMyzbDwBGbySewcpAl5Qh7IulJBbN82Zu1QhvFgXhxgExiyH5uBVeEeJCzFL3aS5zNwBLdLa7rrIG2wRvjR%2B0DMK21ZVQA%2BtGZ2cqreUjlDj3dXxmC8h%2Bm1uWwbZhNKcj0Mo7NA4uxNB7dIf89Tt5PXTyTcnFgL0kysfx4nLLKFLMtsHLCtK3DmcQ2F-Nazr7RO4bngm9c5vPYXZUl%2BXBAJ0nyj2x3euIM7rvSS5Hte9I7B%2BwHQPN%2BTrfZrnor5xHGDcn1Jebb4Hw1kw8BMNsqdQRn125VnKHWTv9p72a3LrcfTLGEN0gE%2BfoIr430XPCLmKIeZzQptvQWNNYaAm5G9L%2BjISCwFwAAcTwNjXGlB8anmAWaO%2BYMFItz5veSKedw7vwRsfB2ncAg1kMtAAA1jWNszC8CUGlPtdspE3TS0Jj8TKG9eZ3X5oVChtN946gZiXKMsZWR1lwNuFwptEBkBRowTgBM3KgnErXRugImpEJaiQ0RZDOqwOFmcbkoth5xm2IwH%2BagTz7XoA4zwbwNwhFgCCMEKgXEMRMsIyBW80IWIkXAn43IrZ2KquGLwzhuCnm2HoqSbB1FMC0dRQJTFgnq1QpxcJu9KGAl9M5GhHcTTcHbE6eABAyKwHxjsOR08clyWMTdUx2dzEYQiVY5Ivpi4-EgFg9gwghhEAANKIEQCIXxQwQSsNUB7NpU1wHEM3qQsJvTimSL9EPYZDDVBMJNJQWOsAhqS0CF5eAk9HYBCwRuHBBMDE-CMaTDZIjunbLDns0pR9Dmtj0huEgydoAcFqvE4Iv5NGnjIBQL2ATjK5PvmrKBPz%2B4F0wL6T%2BwzRkQomVgJUg0aJvBEEqMIKpkXtI%2BSYzZZj8QwL6RbAZiDhnTNmZtGuaS5RHMYCc1kryXq-DAWAGawcn7tXEbsyJZxfTUOGcS7wuA6QIFNICIRqK1jIjhLcSgxh8b7CEWiIAA)

#### Code

```fsharp
        let allDecls = inheritsL @ iimplsLs @ ctorLs @ instanceValLs @ methLs @ ilFieldsL @ propLs @ eventLs @ staticValLs @ nestedTypeLs

        let allDecls = inheritsL + iimplsLs + ctorLs + instanceValLs + methLs + ilFieldsL + propLs + eventLs + staticValLs + nestedTypeLs

```
            

#### Result

```fsharp
let allDecls =
    inheritsL
    @ iimplsLs
      @ ctorLs
        @ instanceValLs
          @ methLs @ ilFieldsL @ propLs @ eventLs @ staticValLs @ nestedTypeLs

let allDecls =
    inheritsL
    + iimplsLs
    + ctorLs
    + instanceValLs
    + methLs
    + ilFieldsL
    + propLs
    + eventLs
    + staticValLs
    + nestedTypeLs

```
            
#### Problem description

Fantomas has heurstic that if the operators are all the same in a large nested infix application, it will place them all on a new line. For some reason, this isn't working for `@`.

#### Extra information

- [ ] The formatted result breaks my code.
- [ ] The formatted result gives compiler warnings.
- [ ] I or my company would be willing to help fix this.

#### Options

Fantomas master branch at 2022-09-01T05:33:17Z - 582e9ab5dc7d7e041e5f4d1ce203dc40b98d98a7

Default Fantomas configuration

<sub>Did you know that you can ignore files when formatting from fantomas-tool or the FAKE targets by using a [.fantomasignore file](https://github.com/fsprojects/fantomas/blob/master/docs/Documentation.md#ignore-files-fantomasignore)?</sub>

**Comment by nojaf** [comment]
The problem isn't limited to `@`, sample with [^^](https://fsprojects.github.io/fantomas-tools/#/fantomas/preview?data=N4KABGBEDGD2AmBTSAuKEOaxALtAMgDoBOhAdtpRgHrVgDOiADvmABQ4CGA5gAoCuZaDn6ccAS1gVCkADwyAlCXJUqtduIBmYTvHgBRAB45inAMpNO0RGBwALRBTPN8AOnqXrYRABtG3gFsmHABPfCVSClVsdTYfThDYfhwAFRCmRAB1cXsASTJNWABBMnheYkRoMCQyADdvOrAAFh1ibjSIlWisdWJxbjscVg4eASERMUkyGQA%2BRUgAGnAoRhwJMm56VDAAbWWIUGxIABJQjO3IcTIcRf2ME9rOH35kND2qAAYlqkh8mpwzOIAF7Ib6UJp3CAAXTuAF8wQdIVBTulXlArjcEfdjo9nmj3pQAIxYzCQACynEM%2BCuiHwjm49luVEJACYPkiYZh4XdDlgTmc0ZBHPAAPKaalkRBmUI%2BUFIh5PF7bAnYFkk%2B76UpiiVyn4%2BTSQDlwrG80ko85oSAAI1gsB8TL5OMV%2BKREAAzOqIL8yIxiDgAGJXJ4AOUQAHcfDSHdgTC8jVyTfLzYKbXbo9jcUq3q7mp6oBYrIgAEKIQoVXicUwBRA4RDEdOYWOIeMYbmYU3YgUXVP29UKvHKnMAVjzkAL1hLZdpsDDdegnEY%2BVqsHnEikDYwTZbEDbGA7XuT3dtvaTmZdVAAbKPx8XS7AKgBVJgZYjzxd1FeTdd5zRPRjbsBd0RI5D0tHsN37LNdhzAB2a9PFvKcAGF4noegkKkehY2Ee8NwgX8-GbbBOVbRMQK7MDjwgp0B2zKgAA54MLSd70QMlEACK06zwsACP-YjjR5JMKKgcC%2BxoqCVSwABOJiJzvCoMJ8b8cz4oisBIncyMdETrSo8Sz0HZkvhzMcEKKTRa2IDCAgCTgeK3ASEyE8jUSPNMDOdIyiWJUybxYipnACcQ4GUsgeLUgCgLAfdkV0sTTy8uiiTVPzzMsusgpCu0VKoRyNME9thLcyiPMS2joOZD00sLIpiCSUoABFfHEYKrIc4g4yc0iXJ0kr0WuajDOS7BCSaUcKUMXJNBSBwyDMOx7xwbJ4EZPN2W6rTerNXSMSGpLKqJEdTMm6bZscfRCIWpaVrWnMLw2grnKK1yLQGzFysknNCSvE7KXyTRxEMEUXzEe8jCYCo0KmHj6MezBNMA7Sdv6y5Bs8iqpMwQk4L%2BwwACVKnveBbrsHimnhjBEei2L%2BVRvaMa%2B5lGLxwm4GIeBg34Ti6zFXJawCLY80JKLkc7VGAn4HwJEjSV-XvOy1jrNJzkZ88iVk0y2eJskpZlml5eIRX2tHaA7ErKwrIAfTDcRVrJ0XtvFt60Y%2BkDhsO1UTJ%2BSa6tMEIRWIaksNJ2HKehQq92Kl2Gc%2B9XVV8n3KT9hJA%2BDnAuZ54g%2BYFoXvsdl6%2BpdyXpfEWXEEN43lbctXvNVVKfhTgOg-ELDddL8vK7EE3TLNi3hDrG27cZAuo9ewVY-dg6sYwFlqqTwwADVnSLK54CubhQ7zOHR%2BAouJ-RuO66wFlxrx-1BGEKZV9KDet5zCnd5i6OD7dx0PZniAWWOhfGtgHAADiNYIZQ3oFMe%2BDFw5gGpmLA88V9JHxGifX6Ps9ZlxpEWZS0AADWRZTA4JrPQEU81ODViUtzcKP4-zqQRpHPeKMXYJSnpjHMLJcY-FDBGDBNZZyOBVogZqgMyA5CmCUeA7Es55zyp1GhVM6HP3Hu5E8zCmaUBZCzRukZuBkAvuMKYgJtFiH4BUFIsA-iOC4GuShqlqFP1pqBUSCCVHxxPprTRVkyCTFqNODYEiuLEEEVcERmEIq2M2kjJ2cDUZMPftPHMbpvaUHJGgnU%2BBSFWngJwFCsAwEbE4eXUJhE7EvyUftFhVA3SJySRDOsrULFPAANKIGYOY64%2BQ8GcCEGTKhRTwk0xKaVZRsTymUDdA3JJmDOnYJ1PQOqDVOaICwogcRaDy4gMWWAkJeZ8q0OemPfepTa5IMwG6eeEzKwBQEa3aAfRgqeNrPAB8wipDNWgPEUwVjCn8Sej1QuDCUxOOGao7Aboz4-Bqbc%2BpPhpT1X4FhTqTBpQhFlF82REc9n0Odq-MpwKsBuh-kkppzBJqZ38WKSZZBpk0ikUSD4UCYGRLitEwFZoP7xJQUkmFIUcBkgQLqSgkU%2BnLChGCS49B-RgO2JFWEQA)

**Comment by nojaf** [comment]
Fix available in [v5.2.0-beta-001](https://www.nuget.org/packages/fantomas/5.2.0-beta-001)
